docs/getting_started.md

```diff
@@ -0,0 +1,128 @@
+
+## Install Prommis
+ProMMiS is distributed as a Python package, so can be installed through Python's standard tool, `pip`. These instructions assume you can run commands from a Mac OSX, Linux, or Windows Powershell terminal.
+
+### Create Python environment (optional)
+Before installing the PrOMMiS software, we recommend you set up "environment" that will let you install Python packages without affecting your default setup.
+Instructions are given below for doing this using Conda and Miniforge, but there are other methods such as [pyenv](https://github.com/pyenv/pyenv) and [virtualenv](https://virtualenv.pypa.io/en/latest/) that have similar effects.
+
+### Create Conda / Miniforge environment
+
+Conda is the name for the packaging environment created by the Anaconda corporation.
+Miniforge is software that installs Python packages, and sets up
+Python environments, without commercial and licensing concerns associated
+with the Anaconda software and package libraries.
+
+Long story short, install miniforge from these links:
+
+- [Home page](https://github.com/conda-forge/miniforge)
+- [Releases](https://github.com/conda-forge/miniforge/releases)
+
+This will enable the `conda` command you can use to install a new Python
+environment:
+
+```
+conda create -n prommis python=3.11 -y
+```
+
+Before you do any work you need to "activate" the environment:
+```
+conda activate prommis
+```
+
+### Install PrOMMiS
+
+We will now install the PrOMMiS Python package into this environment.
+```
+pip install prommis
+```
+This will take a couple of minutes.
+Once it is done, you can continue to try an example.
+
+### Install IDAES "extensions"
+
+After installing ProMMiS, you will need to install binary packages for the
+solvers. These are contained in the IDAES package and called "extensions".
+The following command can be used to automatically install the IDAES extensions:
+
+```
+idaes get-extensions
+```
+
+:::{note}
+Depending on your operating system, additional steps might be needed. For more information, refer to the [IDAES installation guide](https://idaes-pse.readthedocs.io/en/stable/tutorials/getting_started/index.html).
+:::
+
+
+## Try an example
+This section fetches and runs a Jupyter Notebook containing a PrOMMiS flowsheet.
+
+We start by creating a working directory, then download the notebook from GitHub,
+then -- after making sure the Jupyter Notebook code is installed -- open that
+notebook in Jupyter.
+
+### Create subdirectory to work in
+
+```
+mkdir prommis-work
+cd prommis-work
+```
+
+### Download the UKy flowsheet Jupyter Notebook
+
+::::{tab-set}
+
+:::{tab-item} Console (curl)
+Run the following command using the [curl](https://curl.se/) command-line tool.
+```
+curl -o uky_flowsheet-solution.ipynb https://raw.githubusercontent.com/prommis/prommis/refs/heads/main/docs/tutorials/uky_flowsheet-solution.ipynb
```

Comment:
You also need the following for the flowsheet diagram:

`curl -o uky_flowsheet.png https://raw.githubusercontent.com/prommis/prommis/refs/heads/main/docs/tutorials/uky_flowsheet.png`